kratos/sources/constitutive_law.cpp

```diff
@@ -644,23 +644,27 @@ void ConstitutiveLaw::CalculateStressResponse (Parameters& rValues, Vector& rInt
  */
 void ConstitutiveLaw::InitializeMaterialResponse(Parameters& rValues,const StressMeasure& rStressMeasure)
 {
+    if (!RequiresInitializeMaterialResponse()) {

```

Comment:
this should be removed

Reply:
Idem

Reply:
Why, because of the warning?

Reply:
first of all yes, but more important bcs these macros basically serialize the code, since the counter for the macros is shared